src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/AcknowledgeClaimCallbackHandler.java

```diff
@@ -121,10 +116,11 @@ private CallbackResponse populateRespondentCopyObjects(CallbackParams callbackPa
         if (solicitorRepresentsOnlyOneOrBothRespondents(callbackParams, RESPONDENTSOLICITORTWO)) {
             isRespondent1 = NO;
         }
-        updatedCaseData.solicitorReferencesCopy(caseData.getSolicitorReferences());
-        updatedCaseData.isRespondent1(isRespondent1);
+        var caseDateBuilder = caseData.toBuilder();
```

Comment:
Can you not do it as a one liner? caseData.toBuilder().solicitorReferencesCopy(etc).etc().build()